compclub/website/urls.py

```diff
@@ -1,10 +1,16 @@
-from django.urls import include, path
+from django.conf.urls import re_path
+from django.contrib.auth import views as auth_views
+from django.urls import path
 
 from . import views
 
+app_name = 'website'
+
 urlpatterns = [
+    re_path(r'^login/$', auth_views.login, {'redirect_authenticated_user': True}, name='login'),
+    re_path(r'^logout/$', auth_views.logout, name='logout'),
```

Comment:
Is there a reason why we're using `re_path` instead of `path`?

Reply:
yup we should be using `path` i mixed up both